include/boost/geometry/extensions/gis/projections/proj/hatano.hpp

```diff
@@ -49,8 +49,8 @@ namespace boost { namespace geometry { namespace projections
             static const int NITER = 20;
             static const double EPS = 1e-7;
             static const double ONETOL = 1.000001;
-            static const double CN = 2.67595;
-            static const double CS = 2.43763;
+            static const double CN_ = 2.67595;
+            static const double CS_ = 2.43763;
             static const double RCN = 0.37369906014686373063;
```

Comment:
Ok, then I suggest to use lower-case letters for those names to avoid problems with macros and be in line with the Boost guidelines.

Btw, where are those numbers from? They look like single-precision, probably not good enough for double or more precise type.